source/tests/python/rusage_tests.py

```diff
@@ -0,0 +1,252 @@
+#!@PYTHON_EXECUTABLE@
```

Comment:
Also in `^/timemory/tests` directory. Remove any existing files from tests directory